hasher-matcher-actioner/webapp/src/pages/Matches.jsx

```diff
@@ -0,0 +1,278 @@
+              htmlSize="40"
+            />
+            <InputGroup.Append>
+              <Button onClick={loadSearchPage}>Search</Button>
+            </InputGroup.Append>
+          </InputGroup>
+        </Row>
+        <Row className="justify-content-between mt-3">
+          <Col md="5">
+            <Form.Check
+              id="match-list-filter-radio-contentid"
+              type="radio"
+              value="contentId"
+              label="Content ID"
+              checked={localSearchInAttribute === 'contentId'}
+              onChange={e => setLocalSearchInAttribute(e.target.value)}
+            />
+          </Col>
+          <Col md="4">
+            <Form.Check
+              id="match-list-filter-radio-signalid"
+              type="radio"
+              value="signalId"
+              label="Signal ID"
+              checked={localSearchInAttribute === 'signalId'}
+              onChange={e => setLocalSearchInAttribute(e.target.value)}
+            />
+          </Col>
+          <Col md="3" className="push-right">
+            <Button
+              variant="light"
+              size="sm"
+              onClick={() => {
+                setLocalSearchQuery('');
+                inputRef.current.focus();
+              }}>
+              Clear
+            </Button>
+          </Col>
+        </Row>
+      </Form.Group>
+    </Form>
+  );
+}
+
+MatchListFilters.propTypes = {
+  searchInAttribute: PropTypes.oneOf(['contentId', 'signalId']),
+  searchQuery: PropTypes.string,
+};
+
+MatchListFilters.defaultProps = {
+  searchInAttribute: 'contentId',
+  searchQuery: undefined,
+};
+
+function MatchList({selection, onSelect, searchInAttribute, searchQuery}) {
+  const [matchesData, setMatchesData] = useState(null);
+
+  useEffect(() => {
+    let apiPromise;
+    if (searchInAttribute === 'contentId') {
+      apiPromise = fetchMatchesFromContent(searchQuery);
+    } else if (searchInAttribute === 'signalId') {
+      /** Hack alert. This expects strings like
+       * "facebook/threatexchange|12312121" to be the searchQuery. A pipe
+       * separates the source from the signal id */
+      const parts = searchQuery.split('|');
+      apiPromise = fetchMatchesFromSignal(parts[0], parts[1]);
+    } else {
+      apiPromise = fetchAllMatches();
+    }
+
+    apiPromise.then(matches => setMatchesData(matches.match_summaries));
+    // .catch(err => console.log(err));
+  }, [searchInAttribute, searchQuery]);
+
+  return (
+    <>
+      <Spinner hidden={matchesData !== null} animation="border" role="status">
+        <span className="sr-only">Loading...</span>
+      </Spinner>
+      <Collapse in={matchesData !== null}>
+        <Row className="mt-3">
+          <Col>
+            <table className="table table-hover small">
+              <thead>
+                {/* TODO: Undecided nomenclature */}
+                <tr>
+                  <th>Content Id</th>
+                  <th>Matched in Dataset</th>
```

Comment:
Can we change this to Source for now so it matches the other pages (even if we want to change it later when we're more decided on nomenclature etc)?